SharePointFramework/ProjectWebParts/src/components/ProjectPhases/ChangePhaseDialog/index.tsx

```diff
@@ -53,9 +56,12 @@ export const ChangePhaseDialog = () => {
         modalProps={{ isDarkOverlay: true, isBlocking: false }}
         onDismiss={() => context.dispatch(DISMISS_CHANGE_PHASE_DIALOG())}>
         {state.view === View.Confirm && context.props.useDynamicHomepage &&
-          <div className={styles.useDynamicHomepageContent}>
```

Comment:
Miiiiinor, but I think the class name `useDynamicHomepageContent` can be a bit confusing. It almost sounds like a react hook. https://reactjs.org/docs/hooks-overview.html